app/models/provider_interface/provider_applications_filter.rb

```diff
@@ -101,6 +101,31 @@ def status_filter
       }
     end
 
+    def course_type_filter
+      return unless FeatureFlag.active?(:teacher_degree_apprenticeship)
+
+      postgraduate_courses_param_name = 'postgraduate_courses'
+      teacher_degree_apprenticeship_param_name = 'teacher_degree_apprenticeship_courses'
+
+      {
+        type: :checkboxes,
+        heading: 'Course type',
```

Comment:
We are not consistent in this, but I would prefer that text on the page is in locales. (Course type, Postgraduate courses, 'Teaching degree apprenticeship (TDA) courses'). Not a blocker, but it is weird to have to look in so many places for text on the page.

Reply:
Done :)